rhel8/profiles/abcd-low-inline.profile

```diff
@@ -0,0 +1,10 @@
+documentation_complete: true
+title: ABCD Low for Red Hat Enterprise Linux 8
+description: >-
+  This profile contains configuration checks that align to
+  the ABCD benchmark.
+selections:
+  - security_patches_up_to_date
+  - abcd:R1
+  - abcd:R2
+  - abcd:R3
```

Comment:
I don't have a strong opinion, but I have slight preference on this one.

Reply:
What I like about it is that it allows to keep overrides of unwanted selections close to the control that introduces them, and it gives user greater freedom with position of security control selections in the file, especially if they come from more than one policy.

Reply:
The colon looks fine for now.